.github/workflows/daily-qa.yml

```diff
@@ -0,0 +1,65 @@
+# This workflow runs our daily scheduled acceptance tests. It runs each test suite against the
+# development head (e.g. main) and all officially supported branches.
+#
+# As this is meant to be run via scheduling, the workflow itself only runs on the default branch
+# (e.g. main), so any changes you make will affect any other branches we test via this workflow.
+name: Daily tests
+
+on:
+  workflow_dispatch: { }
+  schedule:
+    # Runs at 01:00 every day; see this link for more: https://crontab.guru/#0_1_*_*_*
+    - cron: '0 1 * * *'
+
+env:
+  # This URL is used as the business key for the various QA workflows
+  BUILD_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+
+jobs:
+  # This job will figure out the last supported versions based on the latest tags.
+  #
+  # Versions around found by sorting the tags, removing any that doesn't match semantic version,
+  # and ignoring those with suffixes, and sort in natural order reversed. To get the second-highest
+  # minor branch, we do the same, but we also filter out any versions which match the minor version
+  # of the latest stable version.
+  get-versions:
+    name: Compute branch matrix
+    runs-on: ubuntu-latest
+    outputs:
+      latest-version: ${{ env.LATEST_VERSION }}
+      previous-latest-version: ${{ env.PREVIOUS_LATEST_VERSION }}
+    steps:
+      - uses: actions/checkout@v3
+      - run: git fetch --tags
+      - run: |
+          latest=$(git tag | grep '^[0-9]\.[0-9]\.[0-9]$' | sort -Vr | head -n1)
```

Comment:
:x: I think we need `'^[0-9]\.[0-9]+\.[0-9]$'` otherwise we can't find 8.11.1 or?

Reply:
Oh yeah, good catch, we need to match multiple digits between `.` :man_facepalming: